docs/latest/third-party-functionality/imports.md

```diff
@@ -1,9 +1,9 @@
 ## Imports {#imports}
 
-Both core and registered functionality can be retrieved by the top-level `import` function, e.g.:
+Both core and registered functionality can be retrieved by the top-level `importInternalModule` function, e.g.:
```

Comment:
"core"

Maybe, helpful internal library methods and utilities can be retrieved by...

You can add to the list of methods that can be retrieved by...

Reply:
I like it 👍

Reply:
Are you marking as resolved to say that the change has been actioned?